scipy/stats/tests/test_levy_stable_fitstart.py

```diff
@@ -0,0 +1,91 @@
+# Test scipy.stats.levy_stable._fitstart().
+
+import numpy as np
+import pytest
+
+from scipy.stats import levy_stable
+from scipy.stats import rv_continuous as rvc
+
+class TestLevy_stable:
+    def test_neg_beta(self):
+        # Confirm that _fitstart() gets same estimate of alpha for +beta and -beta.
+        rvc.random_state = 2021
+        NPOINTS = 1001
+        ALPHA = 1.5
+        BETA_POS = 1
+        
+        draw = levy_stable.rvs(alpha=ALPHA, beta=BETA_POS, size=NPOINTS)
+        # Fit stable distribution to positive and negative mirror image data, examine alpha.
+        alpha_pos, alpha_neg = [levy_stable._fitstart(data)[0] for data in [draw, -draw]]
+        
+        assert np.isclose(alpha_pos, alpha_neg)
+        
+    def test_neg_delta(self):
+        # Confirm that _fitstart() returns symmetric +delta and -delta if beta flips.
+        rvc.random_state = 2021
+        NPOINTS = 1001
+        ALPHA = 1.5
+        BETA_POS = 1
```

Comment:
here and elsewhere, I'm not sure much is gained by defining the variables like this and then using them only once vs. just using the values directly in the keyword arguments

Reply:
Fair point, thanks.